misc/config_tools/static_allocators/gpa.py

```diff
@@ -297,8 +297,8 @@ def get_pt_devs_io_port_passthrough(board_etree, scenario_etree):
     return dev_list
 
 def get_pci_hole_native(board_etree):
-    resources = board_etree.xpath(f"//bus[@type = 'pci']/device[@address]/resource[@type = 'memory' and @len != '0x0']")
-    resources_hostbridge =  board_etree.xpath("//bus[@address = '0x0']/resource[@type = 'memory' and @len != '0x0' and not(starts-with(@id, 'bar')) and not(@width)]")
+    resources = board_etree.xpath(f"//bus[@type = 'pci']/device[@address]/resource[@type = 'memory']")
+    resources_hostbridge =  board_etree.xpath("//bus[@address = '0x0']/resource[@type = 'memory' and not(starts-with(@id, 'bar')) and not(@width)]")
```

Comment:
Why do we want to skip `@len != '0x0'` here? To get the resources assigned to all bridges/switches, it is the `@address = '0x0'` check that should be removed.

Reply:
resolved.